scripts/openstack-quickstart-demosetup

```diff
@@ -696,7 +696,8 @@ if [ "x$with_manila" = "xyes" ]; then
     start_and_enable_service openstack-manila-scheduler
     start_and_enable_service openstack-manila-share
     start_and_enable_service openstack-manila-data
-
+    # wait a bit until manila-api is ready
+    sleep 5
```

Comment:
not particularly happy about sleeps.. can we add a small loop around manila type-list ?